@types/frame/state.d.ts

```diff
@@ -174,3 +174,5 @@ interface SmartAccount {
   apps: any,
   dao: any
 }
+
+type FrameWindow = BrowserWindow & { frameId: string }
```

Comment:
I think best practice is usually to locate types where they are defined or centrally referenced/managed as much as possible. the only reason I created this global .d.ts file for state is because the store isnt typed and we access it everywhere in the app right now